rust/sealevel/rust-toolchain

```diff
@@ -1 +1,3 @@
-1.60.0
+[toolchain]
```

Comment:
I believe we can remove this file now
Iiuc it's actually ignored entirely by `cargo build-sbf`, and because we have `rust-toolchain` in the parent directory with the same contents, this is redundant